### PR TITLE
Add dropdowns

### PR DESCRIPTION
**Pull request by dsdemaria**
From issue #18 – Implements the Dropdown component from the component library (bells and whistles included).

### CODE REVIEW

**Comment by JohnTasto** [comment]
Edit: somehow I was still looking at the build logs for #26 when I posted this comment. I was wondering how two PRs could cause Travis to fail in the same way...

**Comment by JohnTasto** [comment]
This is the correct error:
```
$ yarn
yarn install v0.23.2
error An unexpected error occurred: "Unknown token 4832:1 in /home/travis/build/hackoregon/housing-frontend/yarn.lock".
info If you think this is a bug, please open a bug report with the information provided in "/home/travis/build/hackoregon/housing-frontend/yarn-error.log".
info Visit https://yarnpkg.com/en/docs/cli/install for documentation about this command.
```

There is still an unresolved merge conflict in yarn.lock on line 4832:
```
<<<<<<< HEAD
react-input-autosize@^1.1.0:
  version "1.1.0"
  resolved "https://registry.yarnpkg.com/react-input-autosize/-/react-input-autosize-1.1.0.tgz#3fe1ac832387d8abab85f6051ceab1c9e5570853"

react-leaflet@^1.1.4:
=======
react-leaflet@^1.1.4, react-leaflet@^1.1.6:
>>>>>>> upstream/master
```

File: src/state/parameters/reducer.js [hunk 1]
```diff
@@ -23,12 +26,18 @@ const other = (state = INITIAL_OTHER_STATE, action) => {
...
+        },
       };
     case actionTypes.UPDATE_OTHER_UNIT_SIZE:
       return {
         ...state,
-        unitSize: action.payload,
+        unitSize: {
+          label: action.payload.label,
+          value: action.payload.value,
+        },
```
Comment (nmuldavin) [inline src/state/parameters/reducer.js:40]: Why doesn't 

```
unitSize: action.payload
```

work here?

File: package.json [hunk 2]
```diff
@@ -38,7 +38,7 @@
     "show:coverage": "npm run test:cover && open coverage/index.html"
   },
   "dependencies": {
-    "@hackoregon/component-library": "0.0.6",
+    "@hackoregon/component-library": "^0.2.0",
```
Comment (nmuldavin) [inline package.json:41]: When you resubmit this should be up to date already

File: src/components/App/index.js [hunk 3]
```diff
@@ -60,23 +61,23 @@ export function App({
...
-      <select value={otherUnitSize} onChange={event => setOtherUnitSize(event.target.value)}>
-        {UNIT_SIZES_AFFORDABILITY.map(size => (
-          <option value={size} key={size}>{size}</option>
-        ))}
-      </select>
+      <Dropdown
+        value={otherUnitSize}
+        onChange={event => setOtherUnitSize(event)}
+        options={UNIT_SIZES_AFFORDABILITY}
+      />
```
Comment (nmuldavin) [inline src/components/App/index.js:74]: After #39 there will be just one dropdown for both . . . not sure how they'll be arranged but we'll figure that out later.

**Review by nmuldavin — CHANGES_REQUESTED** [review]
Can you bring this up to date with recent changes, either by copying over to a new branch or rebasing? I left a few comments

**Pull request closed without merging**